en/flight_controller/intel_aero.md

```diff
@@ -23,8 +23,10 @@ get_aero_version.py
 The instructions here are tested with the following version:
 
 ```
-BIOS_VERSION = Aero-01.00.12_Prod
-OS_VERSION = v01.00.04
+BIOS_VERSION = Aero-01.00.13
+OS_VERSION = Poky Aero (Intel Aero Linux Distro) 1.5.1-dev (pyro)"
```

Comment:
Is the " at the end intentional?

Reply:
Oh is not intentional but is the output that get_aero_version.py prints, so better keep in the documentation until fixed in the Linux image.